### ISSUE TITLE
Uncaught Exception - System Error Connection reset by peer

### ISSUE THREAD

**Issue by bradstimpson** [SGrondin/newque#8]
Not sure what generated this error, as Newque is deployed on publically accessible machine perhaps a scraper/bot hit it:

```
UNCAUGHT EXCEPTION: System Error CONNECTION RESET BY PEER {call: read()}
Raised by primitive operation at file "src/unix/lwt_bytes.ml", line 130, characters 42-84
Called from file "src/unix/lwt_unix.ml", line 489, characters 13-24

UNCAUGHT EXCEPTION: System Error CONNECTION RESET BY PEER {call: read()}
Raised by primitive operation at file "src/unix/lwt_bytes.ml", line 130, characters 42-84
Called from file "src/unix/lwt_unix.ml", line 489, characters 13-24

UNCAUGHT EXCEPTION: System Error CONNECTION RESET BY PEER {call: read()}
Raised by primitive operation at file "src/unix/lwt_bytes.ml", line 130, characters 42-84
Called from file "src/unix/lwt_unix.ml", line 489, characters 13-24

UNCAUGHT EXCEPTION: System Error CONNECTION RESET BY PEER {call: read()}
Raised by primitive operation at file "src/unix/lwt_bytes.ml", line 130, characters 42-84
Called from file "src/unix/lwt_unix.ml", line 489, characters 13-24

UNCAUGHT EXCEPTION: System Error CONNECTION RESET BY PEER {call: read()}
Raised by primitive operation at file "src/unix/lwt_bytes.ml", line 130, characters 42-84
Called from file "src/unix/lwt_unix.ml", line 489, characters 13-24
```

**Comment by SGrondin** [comment]
I *think* this is fixed in 0.0.4. Please let me know if it still occurs. I couldn't reproduce it.

**Comment by SGrondin** [comment]
I ran into this error while developing the ruby gem. It'll be fixed in v0.0.6 https://github.com/newque/newque/commit/a61f518bc4a5202f5756a256d651ae3af6afea06
